tests/docker_noinit/Dockerfile

```
@@ -69,7 +71,9 @@ COPY tests/task_definitions/github_actions/.github /home/testuser/test_project/.
 # Copy Docker Compose files
 COPY tests/task_definitions/docker-compose.yml /home/testuser/test_project/
 COPY tests/task_definitions/compose.yml /home/testuser/test_project/
+COPY tests/task_definitions/turbo_repo /home/testuser/turbo_repo
 RUN chown -R testuser:testuser /home/testuser/test_project
+RUN chown -R testuser:testuser /home/testuser/turbo_repo
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)